src/stream.ts

```diff
@@ -160,11 +155,37 @@ export class WebRTCStream implements Stream {
       if (closed || this.writeClosed) {
         break;
       }
-      let res = buf.subarray();
-      let send_buf = pb.Message.toBinary({ message: buf.subarray() });
-      log.trace(`[stream:${this.id}][${this.stat.direction}] sending message: length: ${res.length} ${res}, encoded through pb as ${send_buf}`);
-      this.channel.send(send_buf);
+      const res = buf.subarray();
+      const msgbuf = pb.Message.toBinary({message: buf.subarray()});
+      const sendbuf = lp.encode.single(msgbuf)
```

Comment:
remind me of C code bases, it's cute, but if I just see `lp` in a code base I cannot say that it's really intuitive to me that this means length prefix o.o